Sources/PointFree/Tasks.swift

```diff
@@ -0,0 +1,249 @@
+import Prelude
+import Styleguide
+
+public func sendWelcomeEmails() -> EitherIO<Error, Prelude.Unit> {
+  let emails = EitherIO(
+    run: concat([
+      Current.database.fetchUsersToWelcome(1)
+        .map(map(welcomeEmail1))
+        .run.parallel,
+      Current.database.fetchUsersToWelcome(2)
+        .map(map(welcomeEmail2))
+        .run.parallel,
+      Current.database.fetchUsersToWelcome(3)
+        .map(map(welcomeEmail3))
+        .run.parallel
+      ])
+    .sequential
+  )
+
+  let delayedSend = send(email:)
+    >>> delay(.milliseconds(200))
+    >>> retry(maxRetries: 3, backoff: { .seconds(10 * $0) })
+
+  return emails
+    .flatMap(map(delayedSend) >>> sequence)
+    .flatMap { results in
+      sendEmail(
+        to: adminEmails,
+        subject: "Welcome emails sent",
+        content: inj1("\(results.count) welcome emails sent")
+      )
+    }
+    .map(const(unit))
+}
+
+// TODO: team callouts
+
+private func prepareWelcomeEmail(to user: Database.User, subject: String, content: View<Database.User>)
+  -> Email {
+
+    return prepareEmail(
+      to: [user.email],
+      subject: subject,
+      content: inj2(content.view(user))
+    )
+}
+
+func welcomeEmailView(_ subject: String, _ content: View<Database.User>) -> View<Database.User> {
+  return simpleEmailLayout(content.map(wrapper)).contramap { user in
+    SimpleEmailLayoutData(
+      user: user,
+      newsletter: .welcomeEmails,
+      title: subject,
+      preheader: "",
+      template: .default,
+      data: user
+    )
+  }
+}
+
+private let wrapper = { view in
+  [
+    emailTable([style(contentTableStyles)], [
+      tr([
+        td([valign(.top)], [
+          div([`class`([Class.padding([.mobile: [.all: 0], .desktop: [.all: 2]])])], view)
+          ])
+        ])
+      ])
+  ]
+}
+
+func welcomeEmail1(_ user: Database.User) -> Email {
+  let subject = "Thanks for signing up to Point-Free!"
+  return prepareWelcomeEmail(
+    to: user,
+    subject: subject,
+    content: welcomeEmailView(subject, welcomeEmail1Content)
+  )
+}
+
+let welcomeEmail1Content = View<Database.User> { user -> [Node] in
+  [
+    markdownBlock(
+      """
+      👋 Howdy!
+
+      It's been a week since you signed up for [Point-Free](\(url(to: .home))). We hope you've learned
+      something new about functional programming, and maybe even introduced it into your codebase!
+
+      We'd love to [have you as a subscriber](\(url(to: .pricing(nil, expand: nil)))), so please let us know if
+      you have any questions. Just reply to this email!
+      """
+    ),
+    user.episodeCreditCount > 0
+      ? markdownBlock(
+        """
+        In the meantime, it looks like you have a **free episode credit**! You can use this to see *any*
+        subscriber-only episode, completely for free! Just visit [our site](\(url(to: .home))), go to an
+        episode, and click the "\(useCreditCTA)" button!
+
+        Users typically spend their credit on these popular episodes:
```

Comment:
whatdya think about something like "Here are some of the top episodes that viewers have chosen to use their credits on:"